etc/config.xml

```diff
@@ -56,6 +56,7 @@ tr.shipping.totals td.amount span.price,
                 <bolt_order_caching>1</bolt_order_caching>
                 <api_emulate_session>1</api_emulate_session>
                 <should_minify_javascript>1</should_minify_javascript>
+                <is_pre_auth>0</is_pre_auth>
```

Comment:
can we make it true by default?

Reply:
@daisy1754

It is deliberately set as false. Two reasons:

1. When set to 0 It actually works both ways, pre-auth and no pre-auth. At least in most of the cases. If not for HUF we wouldn't notice it yet.

2. If the merchant updates the plugin on their own (as Capillus did today) without turning on the pre-auth flag and setting the create order endpoints we loose backward compatibility if the default is true.

Reply:
okay fair

Reply:
@matt-thomason let's make sure for new merchants we instruct to set this to true